test/root/request/test_fight_daylight_shavings_buff.html

```diff
@@ -0,0 +1,29 @@
absolute; z-index: 1; right: 0px; top: 1.2em;'><tr><td>Effect Duration</td><td>10.00 (+10.00)</td></tr><tr><td>Effect</td><td>Burning Tongue</td></tr></table>use</span></a>]</font></td></tr></table></center><p>You notice some extra Meat hidden floating on top of some lava.<center><table><tr><td><img src="/images/itemimages/meat.gif" height=30 width=30 alt="Meat"></td><td valign=center>You gain 42 Meat.</td></tr></table></center><p>Your helmet shoots some lasers at your face.  You smell burning hair.<center><table><tr><td><img class=hand src="/images/itemimages/dsstache6.gif" onClick='eff("b22887e130f6b336da89d33e232f13df");' width=30 height=30 alt="Gull-Wing Moustache" title="Gull-Wing Moustache"></td><td valign=center class=effect>You acquire an effect: <b>Gull-Wing Moustache</b><br>(duration: 11 Adventures)</td></tr></table></center><!--familiarmessage--><center><table><tr><td align=center valign=center><img src="/images/itemimages/puckman.gif" width=30 height=30></td><td valign=center>IVM rubs against your leg, shedding a stray pixel.</td></tr></table></center><center><table class="item" style="float: none" rel="id=8298&s=5&q=0&d=1&g=0&t=1&n=1&m=0&p=0&u=."><tr><td><img src="/images/itemimages/pixel7.gif" alt="yellow pixel" title="yellow pixel" class=hand onClick='descitem(512868873)' ></td><td valign=center class=effect>You acquire an item: <b>yellow pixel</b></td></tr></table></center><p>You gain 8 Strengthliness.<bR>You gain 30 Magicalness.<bR>You gain 5 Chutzpah.<bR><!--familiarmessage--><center><table><tr><td align=center valign=center><img src="/images/itemimages/cccbook.gif" width=30 height=30></td><td valign=center>You remember that the map shows a pirate grave under some lava nearby and find a bottle marking the spot.</td></tr></table></center><center><table class="item" style="float: none" rel="id=787&s=35&q=0&d=1&g=0&t=1&n=1&m=0&p=0&u=b"><tr><td><img src="/images/itemimages/bottle.gif" alt="bottle of rum" title="bottle of rum" class=hand onClick='descitem(124295227)' ></td><td valign=center class=effect>You acquire an item: <b>bottle of rum</b> <font size=1>[<a href="craft.php?mode=cocktail&a=787&pwd=329b071e34c5164426d877eca2e0bca0">mix</a>]</font></td></tr></table></center><p><table><tr><td width=40><img src='/images/itemimages/songboombox.gif' /></td><td align='center'>You catch a line from the song playing on your SongBoom&trade; BoomBox:<br />&quot;And then I see the meat in your pants&quot;</td></tr></table><p><a name="end"></a><p><a href="adventure.php?snarfblat=451">Adventure Again (The Bubblin' Caldera)</a><p><a href="place.php?whichplace=airport_hot">Go back to That 70s Volcano</a></center><script>parent.charpane.location.href="charpane.php";</script></td></tr></table></center></td></tr><tr><td height=4></td></tr></table><table><tr><td>[<a href="/account_combatmacros.php">edit macros</a>]</td></tr></table></center></div></div></div></body><script src="/onfocus.1.js"></script><script src="/combatfilter.1.js"></script></html>
```

Comment:
Probably remove the pwd from this just for peace of mind and consistency, even though it couldn't possibly do anything bad